src/Features/CSharp/Portable/BraceCompletion/CurlyBraceCompletionService.cs

```diff
@@ -102,10 +102,10 @@ public override Task<bool> AllowOverTypeAsync(BraceCompletionContext context, Ca
             {
                 newLineEdit = new TextChange(new TextSpan(closingPoint - 1, 0), Environment.NewLine);
                 textToFormat = originalDocumentText.WithChanges(newLineEdit.Value);
-            }
 
-            // Modify the closing point location to adjust for the newly inserted line.
-            closingPoint += Environment.NewLine.Length;
+                // Modify the closing point location to adjust for the newly inserted line.
+                closingPoint += Environment.NewLine.Length;
```

Comment:
still not happy about using Env.NewLine.

Reply:
this is using doc options in a following commit https://github.com/dotnet/roslyn/pull/48468/commits/d9a327c9019b3e371f3a605dd7a0b5a261290419